main.go

```diff
@@ -39,6 +39,7 @@ import (
 
 	//+kubebuilder:scaffold:imports
 	consolev1 "github.com/openshift/api/console/v1"
+	consolev1alpha1 "github.com/openshift/api/console/v1alpha1"
```

Comment:
do we really need 2 versions of the console can not it be done with one.

Reply:
Yes at least for now. As dynamic plugins API are still in alpha and have not made it as part of v1 stable.

Reply:
Good to know it's at least not our fault, I guess? :joy: